src/screens/consul/ConsulIndexScreen.js

```diff
@@ -113,7 +113,7 @@ export const ConsulIndexScreen = ({ navigation, route }) => {
 
   if (isError) showRegistrationFailAlert(navigation);
 
-  if (!Component) return null;
+  if (!Component || !listItems) return null;
```

Comment:
what about empty screen component here?

Reply:
i thought about it. we have `ListEmptyComponent` inside each of our lists. we show it here. in the current situation, an alert appears on the screen and when the button is pressed, it returns to the previous page.

Reply:
ok